### PR TITLE
Add automated app deployment

### PR DESCRIPTION
**Pull request by michaeladda**
depends on binaris/precious#444

also pushed as feature-reshuffle-play to gain the new precious reshuffle play code.
CI is at https://ci.binaris.io/job/gobinaris/job/builder-function/job/feature-reshuffle-play/

### CODE REVIEW

File: builder-function.Dockerfile.j2 [hunk 1]
```diff
@@ -17,4 +17,4 @@ COPY Makefile .
 # and the app itself
 COPY tsconfig.json ./
 COPY src src
-COPY backend backend
+#COPY backend backend
```
Comment (arikmaor) [inline builder-function.Dockerfile.j2]: You can remove this line, `backend` is the destination directory for `tsc` in this case (pending my PR)

File: builder-function.Dockerfile.j2 [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM {{docker_account}}/installer-base:{{installer_base_tag}}
+
+# install general packages
+ENV S11 https://raw.githubusercontent.com/binaris/711/master
+RUN apt-get update && apt-get install -y curl git vim
```
Comment (arikmaor) [inline builder-function.Dockerfile.j2]: vim?

File: builder-function.Dockerfile.j2 [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+RUN curl -sL $S11/ubuntu/node/10.x.sh | bash
```
Comment (arikmaor) [inline builder-function.Dockerfile.j2]: why not 12?

**Comment by michaeladda** [comment]
merging. full success available at https://ci.binaris.io/blue/organizations/jenkins/gobinaris%2Fprecious/detail/feature-reshuffle-play/11/pipeline/88 including builder-function deploy

**Comment by arikmaor** [comment]
@michaeladda My comments were not addressed

File: builder-function.Dockerfile.j2 (hunk 1 above)
Comment (arikmaor) [reply builder-function.Dockerfile.j2]: @michaeladda already removed in #3

File: builder-function.Dockerfile.j2 (hunk 2 above)
Comment (arikmaor) [reply builder-function.Dockerfile.j2]: @michaeladda Can I remove?

File: builder-function.Dockerfile.j2 (hunk 3 above)
Comment (arikmaor) [reply builder-function.Dockerfile.j2]: @michaeladda I assume `node 12`, please verify that this is a mistake so I can update or take the appropriate measures to make sure `node 10` is not a problem

**Pull request merged**